.github/workflows/pr-checks.yaml

```diff
@@ -165,7 +165,7 @@ jobs:
       - name: Run Operator Upgrade
         shell: bash
         run: |
-          ./tests/run-e2e.sh --ci
+          ./tests/run-e2e.sh
```

Comment:
Lets preserve `--ci` since the flag can be used to decide on other things than run `container rm` later.